scripts/GenerateFileCatalog.ps1

```diff
@@ -0,0 +1,53 @@
+[CmdletBinding()]
```

Comment:
It's not required, but it might be helpful going forward if we add to the top of the file comments providing a high-level description of the script.